airflow/contrib/operators/gcs_to_bq.py

```diff
@@ -24,6 +24,11 @@
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 
+try:
+    basestring
```

Comment:
Is there any reason why you have not used the `six` package here? Using this package makes it easier to track where we have the code for the older version of Python. We plan to drop support for the older version, so we will simply delete all references to the `six` package.

Reply:
@mik-laj Thanks for the suggestion. I've updated my code.